libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -118,7 +118,7 @@ std::string IRGenerator::generate(
 		);
 	};
 
-	Whiskers t(R"(
+	Whiskers t(R"(<?isEthdebugEnabled>/// ethdebug: enabled</isEthdebugEnabled>
```

Comment:
I just added these comments for testing purposes. It is not really needed later.

Reply:
I was about to comment: not sure why we'd need to keep these, but fair enough then :-).

Reply:
I would like to keep this, so that it's visible in the tests that ethdebug was enabled. This is basically just a placeholder, in subsequent PRs I would remove this and then the tests would show actual ethdebug data.